src/components/FacetProps.tsx

```diff
@@ -0,0 +1,62 @@
+import { DisplayableFacetOption } from '@yext/search-core';
```

Comment:
nit: is there a reason this is imported directly from core rather than from search-headless-react?

Reply:
nice catch! It was auto generated. Updated